### PR TITLE
RIA-7114 Refactoring + separating for panels to import and share project

### PR DESCRIPTION
**Pull request by AlexanderNikitin-Smartbear**
Merge https://github.com/SmartBear/readyapi-free-os/pull/650 first

https://smartbear.atlassian.net/browse/RIA-7114

### CODE REVIEW

File: src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java [hunk 1]
```diff
@@ -1,88 +1,118 @@
...
+                showCard(repositorySelection.label);
+                shown = true;
+                break;
+            }
+        }
+        if (!shown) {
+            repositorySelections.stream().findFirst().ifPresent(repositorySelection -> {
+                repositorySelection.radioButton.setSelected(true);
+                selectCard(repositorySelection.label);
+                showCard(repositorySelection.label);
```
Comment (antone-sb) [inline src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java]: Move these calls to a separate method as they are also used in the cycle above. And you probably can use jsut repositorySelections.get(0).

File: src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java [hunk 2]
```diff
@@ -1,88 +1,118 @@
...
+    protected void showCard(String label) {
         CardLayout cardLayout = (CardLayout) cards.getLayout();
         cardLayout.show(cards, label);
     }
 
     protected RepositoryForm getSelected() {
         return selected;
     }
+
+    private static String repeat(String s, int count) {
```
Comment (antone-sb) [inline src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java]: Move it to StringUtils.

**Review by antone-sb — CHANGES_REQUESTED** [review]

File: src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java (hunk 1 above)
Comment (AlexanderNikitin-Smartbear) [reply src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java]: Extracted to the separate method

File: src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java (hunk 2 above)
Comment (AlexanderNikitin-Smartbear) [reply src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java]: moved

File: src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java (hunk 1 above)
Comment (AlexanderNikitin-Smartbear) [reply src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java]: This is a common method which uses a result of the `getRepositorySelections()`. But the mentioned method can be overridden and return empty list (hypothetically). So I use streams.

**Review by antone-sb — APPROVED** [review]

File: src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java (hunk 1 above)
Comment (antone-sb) [reply src/main/java/com/smartbear/readyapi/plugin/git/ui/AbstractRepositorySelectionGui.java]: Ok.

**Pull request merged**